automation_tools/__init__.py

```diff
@@ -425,9 +425,24 @@ def setup_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def install_discovery_image():
-    """Task to install foreman discovery image."""
+def setup_foreman_discovery():
+    """Task to setup foreman discovery."""
+
+    admin_password = os.environ.get('ADMIN_PASSWORD')
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    template_url = os.environ.get('PXE_DEFAULT_TEMPLATE_URL')
+    host = env['host']
+    if template_url is None:
+        print('You should specify the PXE global default template URL')
```

Comment:
How should the user provide this information? No clue is given as to which environment variable should be set. Can we drop this check and perform a regular dict lookup above? If the dict lookup fails, a `KeyError` will be raised, and it will include a reference to the required environment variable.